site/docs/latest/getting-started/ConceptsAndArchitecture.md

```diff
@@ -266,13 +266,35 @@ In BookKeeper, *journal* files contain BookKeeper transaction logs. Before makin
 
 A future version of BookKeeper will support *non-persistent messaging* and thus multiple durability modes at the topic level. This will enable you to set the durability mode at the topic level, replacing the `persistent` in topic names with a `non-persistent` indicator.
 
+## Message retention and expiry
+
+By default, Pulsar message {% popover brokers %}:
+
+* immediately delete *all* messages that have been {% popover acknowledged %} by a {% popover consumer %}, and
+* [persistently store](#persistent-storage) all unacknowledged messages in a message backlog.
+
+Pulsar has two features, however, that enable you to override this default behavior:
+
+* Message **retention** enables you to store messages that have been acknowledged by a consumer
```

Comment:
I think one source of confusion here was between using the term consumer. Some people were confused about that if I have multiple subscriptions, all the subscriptions will have to "acknowledge" the messages before the data can be considered for deletion. 

Also, we should add that retention also works when there are no subscriptions to retain the data.

Reply:
@merlimat Is it always the case that all consumers need to acknowledge a message? Or does that vary based on the subscription type?

Reply:
Well, if you have a subscription `s1` with 1 or more consumer that acks everything, this won't necessary mean that the data is going to be deleted. There can be a subscription `s2` where the consumer is disconnected or lagging, and that would keep the data around, until `s2` catches up.